Sources/Workspace/Workspace.swift

```diff
@@ -1262,16 +1266,25 @@ extension Workspace {
         for (specifier, state) in packageStateChanges {
             diagnostics.wrap {
                 switch state {
-                case .added(let requirement):
-                    _ = try clone(specifier: specifier, requirement: requirement)
-                case .updated(let requirement):
-                    _ = try clone(specifier: specifier, requirement: requirement)
-                case .removed:
-                    try remove(specifier: specifier)
-                case .unchanged: break
+                    case .added(let requirement):
+                        _ = try clone(specifier: specifier, requirement: requirement)
+                    case .updated(let requirement):
+                        _ = try clone(specifier: specifier, requirement: requirement)
+                    case .removed:
+                        try remove(specifier: specifier)
+                    case .unchanged:
+                        break
                 }
             }
         }
+        
+        // Get the count of `.unchanged` package state changes.
+        let unchangedCount = packageStateChanges.filter({ if case .unchanged = $0.value { return true }; return false }).count
```

Comment:
This looks very weird. There are two ways you can make this simple.
1. Add a computed property to PackageStateChange enum, `isUnchanged`.
2. Implement Equatable for PackageStateChange enum